### ISSUE TITLE
The wheel always return back to the default selected Value

### ISSUE THREAD

**Issue by divinetouch** [beefe/react-native-picker#11]
When I try to change the value by scrolling the wheel up and down, the wheel always return back to the default selected value.
```
<Picker
                            ref={picker => this.picker = picker}
                            pickerHight={300}
                            showDuration={300}
                            pickerBtnText="Done"
                            pickerCancelBtnText="cancel"
                            pickerData= {['value 1', 'value 2']}
                            selectedValue='value 1'
                            onPickerDone = {pickerValue => console.log(pickerValue)}
                        />
```
From the code above, the wheel always return back to the default selected value of 'value 1'. However, when I click on Done, it does print out the correct value.

**Comment by xwenliang** [comment]
update to newest version 0.3.3

**Comment by divinetouch** [comment]
Thanks for a quick response, however, I'm still seeing the same issue. From the example below, the wheel always scroll back to 2.
```
<Picker
                            ref={picker => this.picker = picker}
                            pickerHeight={300}
                            pickerCancelBtnText='Cancel'
                            pickerBtnText='Done'
                            showDuration={300}
                            pickerData={[1,2,3,4]}
                            selectedValue={2}
                            onPickerDone={(pickedValue) => {
                                console.log(pickedValue);
                            }}
                        />
```
Also as you can see from the screenshot, the word "Cancel" got cut off as well.

![screen shot 2016-01-17 at 12 14 04 pm](https://cloud.githubusercontent.com/assets/4777729/12378944/e26922ac-bd13-11e5-90ee-370646fb80ca.png)

**Comment by xwenliang** [comment]
I just tried your code on my <b>newest version 0.3.3</b>, it was all right except the cancelBtn，since the toolBar was flexbox, and title:confirmBtn:cancelBtn = 4:1:1, you can change the title to 3 or smaller, like this:
```javascript
<Picker
    ref={picker => this.picker = picker}
    pickerHeight={300}
    pickerCancelBtnText='Cancel'
    pickerBtnText='Done'
    showDuration={300}
    pickerData={[1,2,3,4]}
    selectedValue={2}
    onPickerDone={(pickedValue) => {
        console.log(pickedValue);
    }}
    pickerTitleStyle={{
        flex: 3
    }}
/>
```
check your version of react-native-picker?

**Comment by xwenliang** [comment]
@divinetouch 
I'm so sorry, this bug was indeed exist on iOS platform, i have fixed it and new version v0.3.5 is released.

**Comment by divinetouch** [comment]
Thanks for your hard work man! It works perfectly now.

:100:
